tests/io/test_write_obj.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+
+"""unit tests for landlab.io.obj module"""
+import pathlib
+
+import pytest
+
+from landlab import HexModelGrid, RasterModelGrid
+from landlab.io import write_obj
+
+LITTLE_HEX_OBJ = """# landlabgrid
+#
+g landlabgrid
+v 1.0 0.0 0.0
+v 3.0 0.0 0.0
+v 0.0 1.732051 0.0
+v 2.0 1.732051 1.0
+v 4.0 1.732051 0.0
+v 1.0 3.464102 0.0
+v 3.0 3.464102 0.0
+f 4// 1// 2//
+f 4// 3// 1//
+f 5// 4// 2//
+f 6// 3// 4//
+f 7// 4// 5//
+f 7// 6// 4//
+"""
+
+LITTLE_RAST_OBJ = """# landlabgrid
+#
+g landlabgrid
+v 0.0 0.0 0.0
+v 2.0 0.0 0.0
+v 4.0 0.0 0.0
+v 0.0 2.0 0.0
+v 2.0 2.0 1.0
+v 4.0 2.0 0.0
+v 0.0 4.0 0.0
+v 2.0 4.0 0.0
+v 4.0 4.0 0.0
+f 5// 4// 1//
+f 1// 2// 5//
+f 6// 5// 2//
+f 2// 3// 6//
+f 8// 7// 4//
+f 4// 5// 8//
+f 9// 8// 5//
+f 5// 6// 9//
+"""
+
+
+def remove_newline_chars(mystr):
```

Comment:
@gregtucker it looks like you are no longer using this. perhaps remove.

Reply:
done